### ISSUE TITLE
Is it possible to disable ctrl+backspace as "erase all" in the markdown editor?

### ISSUE THREAD

**Issue by igorsantos07** [TandoorRecipes/recipes#3451]
### Tandoor Version

1.5.24

### Setup

Docker / Docker-Compose

### Reverse Proxy

No reverse proxy

### Bug description

Simply put, I'm well used to ctrl+backspace to erase a whole word when I'm typing on the computer. However, that causes the textarea for a step to go EMPTY. I guess this is a "feature" from the markdown editor lib used, instead of something implemented by Tandoor, but I wonder if that could be disabled? It feels it could cause more frustration and confusion than being useful to simply scrape the whole textarea and write it again - at least for PC users, not sure if Mac behaves the same way elsewhere.

**Comment by smilerz** [comment]
The editor is 3rd party

**Comment by igorsantos07** [comment]
Any chance you checked if that feature could be disabled, before closing the issue with 5 words?

I didn't because I didn't want to dive in the sourcecode to discover which library is used, given there's not a list of tech used on the docs so far. I guess that you, as a collaborator, would know better.

**Comment by igorsantos07** [comment]
I searched around and found the editor to be Mavon.

Looking into their board, I found a suggestion on how to override shortcut behavior, which seems simple enough (if edited to also check for the backspace key) to override the annoying ctrl+backspace (a recurrent complaint there as well): https://github.com/hinesboy/mavonEditor/issues/341#issuecomment-2025496043

The main issue dates from 2018 :exploding_head: https://github.com/hinesboy/mavonEditor/issues/283
And turns out, it mentions another issue from Tandoor, which wasn't fast-closed like this one - #2174. So I guess this could remain closed as a dupe - but not because "the editor is 3rd party".
